kairon/shared/data/processor.py

```
@@ -254,6 +254,211 @@
+        for param in params_list:
+            if MongoProcessor.is_slot_in_field(param, slot):
+                return True
+
+        return False
+
+    @staticmethod
+    def is_slot_in_callback_action_config(config: dict, slot: Text):
+        if config.get('dynamic_url_slot_name') == slot:
+            return True
+
+        return False
```

Ruff

Return the condition `config.get("dynamic_url_slot_name") == slot` directly

Replace with `return config.get("dynamic_url_slot_name") == slot`

(SIM103)